consumer/guides/mraa/install.md

```diff
@@ -2,12 +2,22 @@
 title: Guides
 permalink: /documentation/consumer/guides/mraa/install.md.html
 redirect_from: /documentation/ConsumerEdition/guides/mraa/install.md.html
+
 ---
 
 # Installation guide for MRAA library on 96Boards CE
 
 Following instructions provides information about installing the MRAA library as a package and building from source.
 
+## MRAA Overview
+**[MRAA](https://iotdk.intel.com/docs/master/mraa/)** (libmraa) is a C/C++ low level skeleton library for communication on GNU/Linux platforms.
+It binds to Python, Javascript and Java to interface with the I/O on ARM platforms such as the DragonBoard 820c.
+It has a structured and sane API where port names/numbering matches the board that you are on.
+Use of libmraa does not tie you to specific hardware with board detection done at runtime you can create portable code that will work across the supported platforms.
+MRAA make it easier for developers and sensor manufacturers to map their sensors & actuators on top of supported hardware and to allow control of low level communication protocol by high level languages & constructs.
+
+Recent addition to the MRAA library by **[Manivannan Sadhasivam](https://twitter.com/mani_sadhasivam)** and **[Ivan Farkas](https://twitter.com/ifarkas)** enabled and extended MRAA for the DragonBoard 820c.
```

Comment:
No need of this information here. The author information can be found in the MRAA commit history.